### ISSUE TITLE
Update translation for Chinese

### ISSUE THREAD

**Issue by robertparley** [satwikkansal/wtfpython#282]
[EN]
Expected time to finish: 3 weeks ago.
The new version is coming. I already create a new pull request to wtfPython-cn.
If you want to read it right now.
Welcome to https://github.com/robertparley/wtfpython-cn
[CN]
更新的中文版已经向 wtfPython-cn提交了pr
想抢先阅读的小伙伴欢迎访问 https://github.com/robertparley/wtfpython-cn

**Comment by satwikkansal** [comment]
Hi @robertparley thanks for updating the translation.

Let me know once your translation is complete. The ideal scenario would be to get your changes merged in original fork, but if the author of original fork is unavailable, I can like your fork directly.

**Comment by 1033786579** [comment]
您发给我的信件已经收到。我会尽快查看邮件。谢谢!

**Comment by robertparley** [comment]
Finished! I'm waiting for merging.
Thanks for your reply! @satwikkansal

**Comment by satwikkansal** [comment]
Awesome, thanks @robertparley I really appreciate your efforts!

I'll link your fork to the English version page if it doesn't get merged in a week.

btw if you like, you can also add an interactive version, to do that you can simply do this,

$ cd irrelevant
$ python notebook_generator.py

This shall output a jupyter notebook in Chinese if you run it in your repo. You can commit the updated notebook to your repo, create a google colab or any other online notebook provider using your repo link, and then update the interactive link as well.

**Comment by robertparley** [comment]
OK.
Let me try to commit the notebook. 
But I can't access the Google colab now.

**Comment by satwikkansal** [comment]
I see you can also use https://mybinder.org/ , if that's not working either, having a notebook in the repository is okay too.

**Comment by satwikkansal** [comment]
Hi @robertparley, looks like the old repository owner didn't respond, I have pointed the English repo to your fork directly, thanks for your work!

**Comment by 1033786579** [comment]
您发给我的信件已经收到。我会尽快查看邮件。谢谢!

**Comment by robertparley** [comment]
@satwikkansal OK. Thank you!😬
